lib/Controller/ItemApiController.php

```diff
@@ -166,6 +166,22 @@ private function setStarred($isStarred, $feedId, $guidHash)
     }
 
 
+    private function setStarredById($isStarred, $itemId)
```

Comment:
Please use the arguments from coarse to fine.

Reply:
What do you think about the original `setStarred($isStarred, $feedId, $guidHash)` located just above this function? The new alternative function just follows the same convention. If we want to change this convention, I guess it would be better to do it at least file-wise, for consistency's sake.

Reply:
I'm currently in the process of cleaning up all code, and more and more functions are being changed to use this convention. New functions should definitely follow them.